sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -115,7 +116,9 @@
     <!--Settings dialog. This shows centered in the middle of the window-->
     <paper-dialog id="settingsDialog" class="dialog" with-backdrop>
```

Comment:
This is a bit odd to have the settingsDialog here rather than datalab-settings be the entire dialog.